common/main/java/com/couchbase/lite/internal/replicator/AbstractCBLWebSocket.java

```diff
@@ -638,6 +641,16 @@ private void didClose(Throwable error) {
             return;
         }
 
+        else if (error instanceof SSLProtocolException) {
+            closed(C4Constants.ErrorDomain.WEB_SOCKET, C4Socket.WS_STATUS_CLOSE_PROTOCOL_ERROR, null);
+            return;
+        }
+
+        if (error instanceof SSLException) {
```

Comment:
As a suggestion, I would put a log here if it doesn't exist somewhere else that indicates that this exception was transformed since the end result (user transient) is not logically related to the input exception.  This is probably good to do any time an exception is mapped, especially if the result is a code that is just used for its transientness.

Reply:
Discussed.  The logging of the exception at the top of this method is sufficient.